src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import Redux from 'redux';
+import bookReducer from './books/books';
```

Comment:
- [ ] Please import `configureStore` and `combineReducers` directly from redux like that:
`import { configureStore, combineReducers } from 'redux';` and use them directly below.